content/oci/storage.go

```diff
@@ -113,7 +114,14 @@ func (s *Storage) Delete(ctx context.Context, target ocispec.Descriptor) error {
 		return fmt.Errorf("%s: %s: %w", target.Digest, target.MediaType, errdef.ErrInvalidDigest)
 	}
 	targetPath := filepath.Join(s.root, path)
-	return os.Remove(targetPath)
+	err = os.Remove(targetPath)
+	if err != nil {
+		if errors.Is(err, fs.ErrNotExist) {
+			return fmt.Errorf("target to delete is not present in the storage: %w", errdef.ErrNotFound)
```

Comment:
Can you follow the same error message pattern as line 114?

Reply:
changed accordingly,